Content.Server/Cloning/CloningSystem.cs

```diff
@@ -72,7 +72,8 @@ public sealed class CloningSystem : EntitySystem
         [Dependency] private readonly ISerializationManager _serialization = default!;
 
         public readonly Dictionary<MindComponent, EntityUid> ClonesWaitingForMind = new();
-        public const float EasyModeCloningCost = 0.7f;
+        // Frontier: easy mod is actually hard mode now - 0.7f<5f
+        public const float EasyModeCloningCost = 5f;
```

Comment:
Would suggest reverting this and setting the cost multiplier (BiomassRequirementMultiplier) in CloningPodComponent.cs to 5.  Not sure this value is configured to get used on the live servers.
```suggestion
        public const float EasyModeCloningCost = 0.7f;
```